composer.json

```diff
@@ -10,23 +10,25 @@
         }
     ],
     "require": {
-        "php": "^8.0 || ^8.1",
+        "php": "^8.0",
         "php-task/php-task": "^2.0",
-        "symfony/http-kernel": "^5.4 || ^6.0",
-        "symfony/dependency-injection": "^5.4 || ^6.0",
-        "symfony/expression-language": "^5.4 || ^6.0",
-        "symfony/config": "^5.4 || ^6.0",
-        "symfony/console": "^5.4 || ^6.0",
-        "symfony/process": "^5.4 || ^6.0",
+        "symfony/http-kernel": "^5.4 || ^6.0 || ^7.0",
+        "symfony/dependency-injection": "^5.4 || ^6.0 || ^7.0",
+        "symfony/expression-language": "^5.4 || ^6.0 || ^7.0",
+        "symfony/config": "^5.4 || ^6.0 || ^7.0",
+        "symfony/console": "^5.4 || ^6.0 || ^7.0",
+        "symfony/process": "^5.4 || ^6.0 || ^7.0",
         "doctrine/orm": "^2.5.3"
```

Comment:
Upgrade to doctrine/orm 3 can not be done yet as inserting a "Single Entity" via `entityManager->flush($entity);` is not longer supported.